### PR TITLE
Add Bitwig

### LINKED ISSUE #4298: Request: Bitwig icon
**Name:** Bitwig
**Website:** https://www.bitwig.com/
**Alexa rank:** 311,740
  

**Official resources for icon and color:**
  
https://www.bitwig.com/static/web/img/BW_Logo_Header.svg

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
Bitwig

**Issue:** Closes #4298
**Alexa rank:** [~260k](https://www.alexa.com/siteinfo/bitwig.com)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Icon from [SVG](https://www.bitwig.com/static/web/img/BW_Logo_Header.svg) in website header, colour from stylesheet.

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
@PeterShaggyNoble I think I've got this down to 3 decimals by rebuilding using plain circles - could you compare it to source, and update if it's preferred?

```svg
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Bitwig icon</title><path d="M4.15 7.782a1.59 1.59 0 1 1 3.181 0 1.59 1.59 0 0 1-3.181 0zm5.741 1.591a1.59 1.59 0 1 0 0-3.181 1.59 1.59 0 0 0 0 3.181zm4.218 0a1.59 1.59 0 1 0 0-3.181 1.59 1.59 0 0 0 0 3.181zm4.15 0a1.59 1.59 0 1 0 0-3.181 1.59 1.59 0 0 0 0 3.181zM5.741 10.409a1.59 1.59 0 1 0 0 3.181 1.59 1.59 0 0 0 0-3.181zm4.15 0a1.592 1.592 0 1 0 .002 3.184 1.592 1.592 0 0 0-.002-3.184zm4.218 0a1.59 1.59 0 1 0 0 3.181 1.59 1.59 0 0 0 0-3.181zm4.15 0a1.59 1.59 0 1 0 0 3.181 1.59 1.59 0 0 0 0-3.181zm4.15 3.182a1.59 1.59 0 1 0 0-3.181 1.59 1.59 0 0 0 0 3.181zM1.591 10.409a1.591 1.591 0 1 0 0 3.182 1.591 1.591 0 0 0 0-3.182zm4.15 4.218a1.59 1.59 0 1 0 0 3.181 1.59 1.59 0 0 0 0-3.181zm12.518 0a1.59 1.59 0 1 0 0 3.181 1.59 1.59 0 0 0 0-3.181zm4.15 0a1.59 1.59 0 1 0 0 3.181 1.59 1.59 0 0 0 0-3.181zm-20.818 0a1.59 1.59 0 1 0 0 3.181 1.59 1.59 0 0 0 0-3.181z"/></svg>
```

**Comment by PeterShaggyNoble** [comment]
Nice work, @adamrusted, thanks 👍🏻 There was a hairline thin difference in the right edge of the 3rd circle on the second row which I cleaned up. As this is now a collaborative effort, we'll need one of the other @simple-icons/maintainers to throw an eye over it before merging.

**Review by service-paradis — APPROVED** [review]
Good job 🎉 🎧 
Thanks @adamrusted and @PeterShaggyNoble!

**Pull request merged**
